api/image/v1alpha1/nodeimage_types.go

```diff
@@ -55,7 +55,7 @@ type NodeImage struct {
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
 	Spec   NodeImageSpec   `json:"spec,omitempty"`
-	Status NodeImageStatus `json:"status,omitempty"`
+	Status NodeImageStatus `json:"status"`
```

Comment:
i had to change the status to be required in order for it to be updated correctly.
however, now some tests are failing :thinking: